src/cmds/manifest/manifest.ts

```diff
@@ -129,48 +129,50 @@ export async function manifest(argv: YargsArgs): Promise<void> {
   }
 
   // check web3Wallet properties
-  const { bundle, initialPermissions } = pkg.web3Wallet || {};
-  if (bundle?.local) {
-    if (!(await isFile(bundle.local))) {
-      logManifestError(`'bundle.local' does not resolve to a file.`);
+  if (pkg.web3Wallet !== undefined) {
```

Comment:
In a follow-up PR, we should probably create an `else` case for this and log a warning or possibly an error.

Reply:
Btw, what motivated this change?

Reply:
There was a typescript complaint somewhere bubbled down inside.

Reply:
Cool. Issue: #96